addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -321,9 +335,7 @@
                             <t t-if="journal_type == 'purchase'">
```

Comment:
Looks like you only did it for the purchase journals?
What about all the other ones?

Reply:
`account` only has upload buttons on purchase journal
the ones on misc journals come from odoo/enterprise#30815

Reply:
All "Upload" buttons functional:
-links on the kanban record
-buttons on the kanban record
-links in the manage dropdown